pepc.js

```diff
@@ -0,0 +1,53 @@
+const API_ACCESS_STATUSES = {
+    success: '🟢 success',
+    error: '🔴 error',
+    unknown: '⚫️ unknown',
+};
+// Display the feature access status (whether the feature can actually be accessed successfully in the browser)
+function updateAccessStatus(permissionName, accessStatus, message) {
```

Comment:
Nit: it took me a bit of reading to realize `message` was an error message.
```suggestion
function updateAccessStatus(permissionName, accessStatus, `errorMessage`) {
```

Reply:
Sorry for the typo here, thanks for fixing up!